### ISSUE TITLE
"No content" placeholder for Course Progress

### ISSUE THREAD

**Issue by pedro16v** [catalpainternational/canoe#974]
## Overview

On the profile, we show users which courses they have completed and which ones are in progress. When there's no content to fill these sections, it stays weirdly collapsed. The design has a placeholder for that.

## Conditions of Satisfaction

- [ ] Add a placeholder on any of the tabs (In Progress / Completed) when those have no content to display.

## Resources

* Check the prototype [on InVision](https://projects.invisionapp.com/d/main?origin=v7#/console/20251865/450048306/preview?scrollOffset=4389)

| Current | Intended |
| - | - | 
 ![Screenshot 2021-08-10 at 13 28 26](https://user-images.githubusercontent.com/2813677/130073460-3d9377ad-e655-49eb-95b5-0b361108c1c2.png) | ![Screenshot 2021-08-19 at 13 59 28](https://user-images.githubusercontent.com/2813677/130073453-76ac2f22-4a64-40cc-8d7c-dcbe694e1c30.png)

**Comment by pedro16v** [comment]
@Juliana-t we probably need an illustration for this. How should we proceed for now?

**Comment by Juliana-t** [comment]
Yes, that is correct and we need an illustration. As @DanielHeads and @andrewwapling are working on the visual system for core Bero and visual assets for the core product, I would like to ask Andy to coordinate the development of this illustration with Daniel and let @megganeturner  know once that is ready (don't forget, svg please) for her to implement

**Comment by andrewwapling** [comment]
Hi @DanielHeads please see idea that could work for this illustration. I though it could be fun to show a learner stranded on an island perhaps reading a book :) Of course open to other ideas however I though this simple scene could work nicely with our new illustration approach and characters.
![IMG_0483](https://user-images.githubusercontent.com/13824612/131270453-e4bfe7db-7ab2-4455-88b0-3c4a937ad676.png)

**Comment by DanielHeads** [comment]
Thank Andy for sharing the idea... I'll work on this when I finish some
illustrations for JID\, probably next Wednesday.
Best
Daniel Cabezas

Designing solutions
for your business

+67077268113
web.facebook.com/umalulikstudio/

**Comment by andrewwapling** [comment]
Hi @DanielHeads no worries, hope it's helpful and thanks for confirming with me 👍

**Comment by andrewwapling** [comment]
Hi @DanielHeads @Juliana-t @pedro16v please see the new illustration in context for review and approval. Look forward to hearing your feedback.

https://invis.io/52Y02L9YTRF#/450048305_Profile_-_General_-_None_In_Progress

**Comment by pedro16v** [comment]
![Screenshot 2021-09-10 at 18 27 58](https://user-images.githubusercontent.com/2813677/132832665-f63ecdac-1217-4926-81b2-cf502ae79741.png)

**Comment by andrewwapling** [comment]
Hi @pedro16v just checking did you require any additional design support on this issue? Thanks, Andy
